SpatialGDK/Extras/schema/rpc_components.schema

```diff
@@ -18,16 +18,19 @@ type UnrealPackedRPCPayload {
 
 component UnrealClientRPCEndpoint {
     id = 9990;
-    event UnrealRPCPayload unreliable_client_to_server_rpc;
-    event UnrealPackedRPCPayload unreliable_packed_client_to_server_rpc;
-    command Void reliable_server_to_client_rpc(UnrealRPCPayload);
+    // The Endpoint is ready to receive RPC requests
```

Comment:
This comment could be a little more descriptive.
Maybe something like - Set to true when authority is gained, indicating that RPCs can be received.

Reply:
Updated as suggested.